### PR TITLE
ppx_pbt, ppx_deriving_qcheck: deprecate

### PR DESCRIPTION
**Pull request by mseri**
This supersedes https://github.com/ocaml/opam-repository/pull/19962

Signed-off-by: Marcello Seri <marcello.seri@gmail.com>

### CODE REVIEW

File: packages/ppx_deriving_qcheck/ppx_deriving_qcheck.0.1.0/opam [hunk 1]
```diff
@@ -15,14 +15,17 @@ authors: [
...
+  "ocaml" {>= "4.10.2" &  < "4.14"}
   "qcheck" {>= "0.17"}
   "ppxlib" {>= "0.22.0"}
   "alcotest" {with-test & >= "1.4.0"}
 ]
 
 build: ["dune" "build" "-p" name "-j" jobs]
 
+flags: deprecated
+post-message: "This package is deprecated and to be considered experimental and unsupported."
```
Comment (mseri) [inline packages/ppx_deriving_qcheck/ppx_deriving_qcheck.0.1.0/opam:27]:
```suggestion
post-messages: "This package is deprecated and to be considered experimental and unsupported."
```

File: packages/ppx_pbt/ppx_pbt.0.1.0/opam [hunk 2]
```diff
@@ -15,14 +15,16 @@ authors: [
...
+  "ocaml" {>= "4.10.0" & < "4.14"}
   "menhir" {>= "20180523"}
   "qcheck" {>= "0.17"}
   "metaquot" {>= "0.4"}
 ]
 
 build: ["dune" "build" "-p" name "-j" jobs]
 
+flags: deprecated
+post-message: "This package is deprecated and to be considered experimental and unsupported."
```
Comment (mseri) [inline packages/ppx_pbt/ppx_pbt.0.1.0/opam:27]:
```suggestion
post-messages: "This package is deprecated and to be considered experimental and unsupported."
```

File: packages/ppx_pbt/ppx_pbt.0.2.1/opam [hunk 3]
```diff
@@ -25,6 +25,9 @@ depends: [
   "ppx_deriving" {>= "5.0"}
 ]
 
+flags: deprecated
+post-message: "This package is deprecated and to be considered experimental and unsupported."
```
Comment (mseri) [inline packages/ppx_pbt/ppx_pbt.0.2.1/opam:29]:
```suggestion
post-messages: "This package is deprecated and to be considered experimental and unsupported."
```

**Comment by vch9** [comment]
Thank you @mseri, it is fine for me!

**Pull request merged**